components-core/src/main/java/com/airwallex/android/core/AirwallexPlugins.kt

```diff
@@ -32,6 +33,15 @@ object AirwallexPlugins {
             return configuration.environment
         }
 
+    @Suppress("SwallowedException")
+    fun getProvider(paymentMethodType: AvailablePaymentMethodType): ActionComponentProvider<out ActionComponent>? {
+        return try {
```

Comment:
Should we merge this with the `getCardProvider`?

Reply:
Yep I'll get rid of `getCardProvider` and make its usage point to this generic function.